Makefile

```diff
@@ -94,6 +94,7 @@ initialize_db:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
+	rm -rf ./user-media/ ./tmp/
```

Comment:
This will also delete the directories or do we actually want to delete just what's underneath them?

You might also want to consider prefixing the command with `-` see https://www.gnu.org/software/make/manual/html_node/Errors.html#Errors so that make continues even if the rm command fails?

I wonder if this should be a separate target called from initialize_db? It might be useful separately in the future?